marathon_acme/tests/fake_marathon.py

```diff
@@ -44,8 +48,22 @@ def get_event_subscriptions(self):
         return self._event_subscriptions
 
     def add_event_subscription(self, callback_url):
-        assert callback_url not in self._event_subscriptions
-        self._event_subscriptions.append(callback_url)
+        if callback_url not in self._event_subscriptions:
+            self._event_subscriptions.append(callback_url)
+
+    def trigger_event(self, event_type, **kwargs):
+        event = {
+            'eventType': event_type,
+            # FIXME: This isn't quite like Marathon's timestamp.
+            # Differences: this has microseconds instead of milliseconds,
+            # Marathon has a trailing 'Z'.
+            'timestamp': datetime.utcnow().isoformat()
```

Comment:
Problem is that Python's `.strftime()` doesn't have an option for milliseconds.
So the best I could come up with is:
`datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z'`

Can't use isotime there as if microseconds are 0 (quite unlikely) then they aren't added.